### DISCUSSION TITLE
Smooth transistion between Display Manager and Cinnamon

### DISCUSSION THREAD

**Discussion by sonicSMASH** [linuxmint/discussions#670] [Ideas]
Hi! When logging in through LightDM into Cinnamon, it flashes to black for 3 seconds and the brings up the desktop. There should be a desktop drawn on the screen as it is happening  (it already appears on the login screen when selecting the user) instead of changing to black.

**Comment by Secret-chest** [comment]
So fade out the login screen UI, load the desktop and slide the panels in. I like that.
